unlock-app/src/unlockTypes.ts

```diff
@@ -3,29 +3,22 @@
 
 // A bug in eslint causes it to think that this exported enum is "unused". So
 // disable eslint for that declaration until they fix it. TODO: follow up on this.
-/* eslint-disable no-unused-vars */
```

Comment:
While I was in here I took the opportunity to remove these `eslint-disable`s, which are no longer required.